### ISSUE TITLE
v8::ObjectTemplate::Set() with non-primitive values is deprecated

### ISSUE THREAD

**Issue by pgkehle** [strongloop/strong-oracle#67]
I've seen this floating around the internet as a reference to fsevents.  However I am still seeing this set of warnings now that I am using the master release after working on my PR branch for so long.  

These do seem to be warnings, and could potentially be turned off.  However, I imagine they are easy fixes.  

Steps take/configuration:
* Removed node_modules directory entirely to ensure a rebuild of all of the required modules.
* Ubuntu 14.04.1 with latest dist_upgrade installed, and rebooted.  
* uname -a: 

```
4.2.0-36-generic #42~14.04.1-Ubuntu SMP Fri May 13 17:27:22 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

* Instantclient version 12.1 is installed
* node version 6.2.1
* npm version 3.9.6
* Strong Oracle version 1.9.0 is installed (verified by CHANGES.md)

[strong oracle object template warnings.txt](https://github.com/strongloop/strong-oracle/files/305576/strong.oracle.object.template.warnings.txt)

**Comment by raymondfeng** [comment]
Can you check what version of `Nan` do you pull in?

**Comment by pgkehle** [comment]
Looks like version 2.3.5:

```
# NAN ChangeLog

**Version 2.3.5: current Node 6.2.0, Node 12: 0.12.14, Node 10: 0.10.45, iojs: 3.3.1**
```

**Comment by bnoordhuis** [comment]
FWIW, I don't get warnings with node.js v6.2.1 and nan 2.3.5.  Can you `rm -rf node_modules` and install again?

**Comment by pgkehle** [comment]
I can, but as I said in the initial issue, I've done that already.. 

From the attachment: 
```
    1: .node [module.js:568] [pc=0xf147a988b44] (this=0x94ea6d77b9 <an Object with map 0x1a756e622b31>#1#,module=0x1070ec7804e9 <a Module with map 0x1a756e623109>#2#,filename=0x1070ec780471 <String[89]: .../node_modules/strong-oracle/build/Release/oracle_bindings.node>)
```

Does that not imply that there is something in oracle_bindings that is the culprit?  Or is the error too vague such that the error points to a different package being includes?

**Comment by bnoordhuis** [comment]
Yes, it's originating from oracle_bindings.node but it's really a nan issue, one I fixed in https://github.com/nodejs/nan/pull/560 and that was released in nan@2.3.0.

**Comment by pgkehle** [comment]
Ok, so I just blew away the directory, and had success.  It's strange when I did this yesterday and the build failed.  This time for success I had to:

* rm -rf node_modules
* npm install
* rm -rf strong-oracle
* npm install strong-oracle

I'll try a couple more iterations, but the build process should be without this sort of hiccup.  Hopefully it's not just me.  Thanks for the help..
